src/index.html

```diff
@@ -5,9 +5,420 @@
+                class="person__photo"
+              >
+              <h3 class="person__name">
+                Scarlett Hawkins
+              </h3>
+              <p class="person__title">
+                Senior Vice President
+              </p>
+              <p class="person__quote">
+                My soul is illuminated by an unearthly joy, as these beautiful spring morning, which I enjoy with all my heart
+              </p>
+            </div>
+            <div class="person">
+              <img
+                src="./images/people/3.jpg"
+                alt="Max Henry portrait"
+                class="person__photo"
+              >
+              <h3 class="person__name">
+                Max Henry
+              </h3>
+              <p class="person__title">
+                Chief Design Officer
+              </p>
+              <p class="person__quote">
+                When my lovely valley of steam rises and half-day sun is above an impermeable
+              </p>
+            </div>
+          </div>
+        </div>
+      </section>
+      <section id="contacts" class="section">
+        <div class="contacts container">
+          <h2 class="contacts__title">
+            Contact us
+          </h2>
+          <div class="contacts__content">
+            <div class="contacts__info">
+              <div class="contacts__text-block">
+                <p class="contacts__phone-title">
+                  Phone
+                </p>
+                <a href="tel:+1 234 5555-55-55" class="contacts__phone-link">
+                  +1 234 5555-55-55
+                </a>
+                <p class="contacts__email-title">
+                  Email
+                </p>
+                <a href="mailto:hello@miami.com" class="contacts__email-link">
+                  hello@miami.com
+                </a>
+              </div>
+              <div class="contacts__text-block">
+                <p class="contacts__address-title">
+                  Address
+                </p>
+                <p class="contacts__address">
+                  400 first ave. n.<br>
+                  suite 700<br>
+                  Minneapolis, MN 55401
+                </p>
+              </div>
+            </div>
+            <form
+              action="#"
+              method="POST"
+              class="contacts__form form"
+            >
+              <input
+                type="text"
+                name="name"
+                placeholder="Name"
+                class="form__input"
+              >
+              <input
+                type="email"
+                name="email"
+                placeholder="Email"
+                class="form__input"
+              >
+              <textarea
+                name="message"
+                cols="30"
+                rows="7"
+                placeholder="Message"
+                class="form__textarea"
+              >
+              </textarea>
```

Comment:
```suggestion
              <textarea
                name="message"
                cols="30"
                rows="7"
                placeholder="Message"
                class="form__textarea"
             ></textarea>
```
ось так має працювати placeholder і лінтер має пропустити

Reply:
Спрацювало )))